controller/script/simulate-proxy/main.go

```diff
@@ -22,7 +22,99 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-/* A simple script for posting simulated telemetry data to the proxy api */
+/* A simple script for exposing simulated prometheus metrics */
+
+type simulatedProxy struct {
+	sleep time.Duration
+}
+
+type proxyVectors struct {
+	requestTotals      *prom.CounterVec
+	responseTotals     *prom.CounterVec
+	responseDurationMs *prom.HistogramVec
+	responseLatencyMs  *prom.HistogramVec
+}
```

Comment:
assuming we're going to run multiple `simulatedProxy` objects, one for each `/metrics` endpoint, i think it will make sense for each `simulatedProxy` to have its own `proxyVectors`.